app/views/dashboard/_eeo_data.haml

```diff
@@ -4,7 +4,7 @@
     .note_item
       = select_tag :department_ids,
       options_from_collection_for_select(@departments, :id, :name),
-      class: 'dept_select', multiple: true
+      class: 'dept-select', multiple: true
```

Comment:
Our class names generally do, so I'd prefer for this one to for consistency's sake. If you're having problems with that I'll be on Slack.